### PR TITLE
Add NUnit and several mocking libs

### PR DESCRIPTION
**Pull request by nsuthison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,12 @@ and some specific libraries for business in Thailand.
 
 ### Testing Framework/Automated Testing
 - xUnit
+- [NUnit](https://github.com/nunit/nunit)
```
Comment (aaronamm) [inline README.md:30]: Since you are working on this area, could you please add a link to xUnit.
After that, I will merge your PR.

Thanks.

Comment (nsuthison) [reply README.md:30]: Done krub

**Comment by RachaneeSaeng** [comment]
Since I made plenty of changes in my PRs and it causes a lot of conflicts on yours, therefore I did code merging and conflict resolving then commit into the `patch-1` in this repository. You may merge the branch to your PR.

**Comment by aaronamm** [comment]
@nsuthison Could you please fix a conflict and after that I will merge your PR?
Thanks.

**Comment by aaronamm** [comment]
@nsuthison  Thanks for your contribution. I appreciate it.
Merged.

**Pull request merged**